app/app.html

```diff
@@ -3,6 +3,9 @@
   <head>
     <meta charset="utf-8">
     <title>Hearmenow</title>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
Yes we should move it over, right now that was the original design by the Twilio package guys